docs/variants/qemu_q35/building-manual.md

```diff
@@ -50,17 +50,12 @@ Following is an example of SMM feature enabled,
  Check the following build command:
 
 ```bash
-build -D DEBUG_ON_SERIAL_PORT -D BOOTLOADER=COREBOOT -a IA32 -a X64 -t GCC5 -b DEBUG -p OvmfPkg/OvmfPkgX64.dsc.
+build -a IA32 -a X64 -t GCC5 -b DEBUG -p OvmfPkg/OvmfPkgX64.dsc.
```

Comment:
```suggestion
build -a IA32 -a X64 -t GCC5 -b RELEASE -p OvmfPkg/OvmfPkgX64.dsc.
```

The build manual should use RELEASE version. The DEBUG version should be used in the development section.

Reply:
Fixed according to the review.